rosdep/base.yaml

```diff
@@ -4549,6 +4549,11 @@ libnss3-dev:
   gentoo: [=dev-libs/nss-3*]
   nixos: [nss]
   ubuntu: [libnss3-dev]
+liboctomap-dev:
+  debian: [liboctomap-dev]
+  fedora: [octomap-devel]
+  rhel: [octomap-devel]
```

Comment:
Should this be removed because it appears to only be on RHEL 8?

https://rhel.pkgs.org/8/epel-x86_64/octomap-devel-1.9.7-1.el8.x86_64.rpm.html

Reply:
No, see my comment in the opening comment:

> Note that I expect CI on this to fail until https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2024-71ae26cb75 lands in EPEL-9, about a week from now.

Reply:
The package was now synced to RHEL-9, so CI finishes.  This is ready to be reviewed and merged.